contentful_rails.gemspec

```diff
@@ -14,10 +14,9 @@ Gem::Specification.new do |s|
   s.description = "A gem to help with hooking Contentful into your Rails application"
   s.license     = "MIT"
 
-  s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
-  s.test_files = Dir["test/**/*"]
+  s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "contentful_model", ">= 0.1.7"
-  s.add_dependency "rails", ">= 4.1.8"
+  s.add_dependency "rails", ">= 4.2", "< 5.1"
```

Comment:
Reason for the change here is that we aren't technically compatible with anything less than Rails 4.2 – I found it when setting up the Travis build.

We use `[ActiveSupport::Concern#class_methods]` in `[ContentfulRails::Sluggable]`, which didn't land in Rails until [4.2](http://apidock.com/rails/ActiveSupport/Concern/class_methods).